releasenotes/notes/expose-tolerances-z2symmetries-9c444a7b1237252e.yaml

```diff
@@ -0,0 +1,36 @@
+---
+features:
+  - |
+    The :class:`~qiskit.opflow.Z2Symmetries` class now exposes the threshold
+    tolerances used to chop small real- and imaginary parts of coefficients.
+    With this one can control how the coefficients of the tapered operator are
+    simplified. For example::
+
+        from qiskit.opflow import Z2Symmetries
+        from qiskit.quantum_info import Pauli
+
+        z2_symmetries = Z2Symmetries(
+            symmetries=[Pauli("IIZI"), Pauli("IZIZ"), Pauli("ZIII")],
+            sq_paulis=[Pauli("IIXI"), Pauli("IIIX"), Pauli("XIII")],
+            sq_list=[1, 0, 3],
+            tapering_values=[1, -1, -1],
+            tol=1e-10,
+        )
+
+    Per default coefficients are chopped with a tolerance of ``tol=1e-14``.
+  - |
+    Add a :meth:`qiskit.quantum_info.SparsePauliOp.chop` method that truncates real and
+    imaginary parts of coefficients individually. This is different to
+    :meth:`qiskit.quantum_info.SparsePauliOp.simplify` which removes a coefficient only if
+    the absolute value is close to 0. For example::
+
+        >>> from qiskit.quantum_info import SparsePauliOp
+        >>> op = SparsePauliOp(["X", "Y", "Z"], coeffs=[1+1e-17j, 1e-17+1j, 1e-17])
+        >>> op.simplify()
+        SparsePauliOp(['X', 'Y'],
+                      coeffs=[1.e+00+1.e-17j, 1.e-17+1.e+00j])
+        >>> op.chop()
+        SparsePauliOp(['X', 'Y'],
+                      coeffs=[1.+0.j, 0.+1.j])
+
+    Note that the chop method does not accumulate the coefficents of the same Paulis.
```

Comment:
I guess you could showcase this explicitly in the example above.

Reply:
You are also not explicitly covering this in the unittest.

Reply:
Added in 029156bf2